packages/react-server-module-tagger/index.js

```diff
@@ -49,81 +8,122 @@ module.exports = function(fullMatch, optString){
  *         { label: 'sub' },
  *         'my-project.src',
  *         'my-project/src/components/my-component.js'
- *     ) // returns "components.my-component"
+ *     ) // returns 'components.my-component'
  * @param  {String}   fn       filename
  * @param  {Object}   opts     { label: 'Optional logger label' }
  * @param  {String}   trim     The leading portion of the name to remove.
  * @param  {String}   basePath The path to the file
  * @return {String}            The logger name, e.g. my-project.components.my-component
  */
-var getName = function(fn, opts, trim, basePath){
-	var slashPattern = isWindows
-		?/\\/g
-		:/\//g
-	var name = fn.substring(basePath.length+trim.length, fn.length)
+var getName = function (fn, opts, trim, basePath) {
+	var slashPattern = isWindows ? /\\/g : /\//g;
+	var name = fn.substring(basePath.length + trim.length, fn.length)
 		.replace(/\.jsx?$/, '')
-		.replace(slashPattern,'.')
+		.replace(slashPattern, '.');
 	if (opts.label) {
-		name += '.'+opts.label
+		name += '.' + opts.label;
 	}
 	return name;
-}
+};
 
 /**
  * Gets isomorphic color objects from filenames.
  * @param  {String} filename
- * @return {Object} An isomorphic color object in the form {"color":{"server":147,"client":"rgb(127,127,212)"}}
+ * @return {Object} An isomorphic color object in the form {'color':{'server':147,'client':'rgb(127,127,212)'}}
  */
-var getColor = (function(){
-
+var getColor = (function () {
```

Comment:
So, given [var hoisting](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#var_hoisting), this isn't actually necessary.  The linting rule would allow this to be at the bottom of the file were it a function declaration, but the simple transformation breaks the closure we use and fails the tests, so I opted for the simplest fix (manually hoisting it), and I'll make this and getName function declarations when I fix #314